wander-embedded-sdk/README.md

```diff
@@ -117,7 +119,168 @@ const wander = new WanderEmbedded({
 });
 ```
 
-### Custom Modal Layouts
+#### Custom CSS Styles
+
+You can add custom CSS styles to the button using `customStyles` option. When using this option, you must use CSS selectors to target specific elements.
+
+Available selectors:
+
+- `:host` - Targets the button container
```

Comment:
These are the ones for the button, but iframe also has some (or should have).

Reply:
@Danziger Its added in this commit: https://github.com/wanderwallet/Wander/pull/741/commits/91ddfbc49b1604e788e48d2b622640f798b26356